pkg/api/apiloader.go

```diff
@@ -251,14 +251,14 @@ func (a *Apiloader) LoadContainerServiceForAgentPoolOnlyCluster(
 
 		// use defaultKubernetesVersion arg if no version was supplied in the request contents
 		if managedCluster.Properties.KubernetesVersion == "" && defaultKubernetesVersion != "" {
-			if !common.AllKubernetesSupportedVersions[defaultKubernetesVersion] {
+			if !common.IsSupportedKubernetesVersion(defaultKubernetesVersion, isUpdate, false) {
```

Comment:
nit: instead of passing in the literal `false` create a `hasWindows` var with the value `false` and pass in `hasWindows` for readability.